pkg/tcpip/header/icmpv4.go

```diff
@@ -49,6 +52,9 @@ const (
 	// icmpv4SequenceOffset is the offset of the sequence field
 	// in a ICMPv4EchoRequest/Reply message.
 	icmpv4SequenceOffset = 6
+	
+	// Milli Seconds in a day calculation for Timestamp reply messages
+    	MsInDay = 60 * 60 * 24 * 1000
```

Comment:
I don’t think you need to export this, and the contraction is confusing.

Can you just move it below to where it is used? E.g.

const millisecondsPerDay = 1000 * 60 * 60 * 24
nowTx := ...